build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017-2018 Daniel Petisme
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import org.gradle.api.tasks.wrapper.Wrapper.DistributionType.ALL
+
+plugins {
+  java
+  id("io.vertx.vertx-plugin") version "0.4.0"
```

Comment:
This version changes the default uber jar classifier from `fat` to `all`. You need to either update all the deployment scripts or configure the classifier. My preference goes to the latter :wink:

Reply:
Yep, embrace the new!